src/Services/AddressService/AddressService.cs

```diff
@@ -216,7 +222,6 @@ private async Task<ProcessRequestEntity> ProcessSearchAddress(
                 {
                     _logger.LogWarning($"{nameof(AddressService)}::{nameof(ProcessSearchAddress)}: Unexpected exception getting address provider {addressElement.Properties.AddressProvider}");
                     throw e;
-                    //throw new ApplicationException($"AddressService::ProcessSearchAddress, An exception has occurred while attempting to perform postcode lookup on Provider '{addressElement.Properties.AddressProvider}' with searchterm '{postcode}' Exception: {e.Message}", e);
```

Comment:
Could we change the name of the caught exception to 'exception' rather than 'e' please